.gitattributes

```diff
@@ -1 +1,2 @@
-dist/** -diff linguist-generated=true
\ No newline at end of file
+dist/** -diff linguist-generated=true
+__tests__/resource/assets/archive-example-test-4.txt eol=lf
```

Comment:
@robinraju this is what you need to fix the test failure on Windows

BTW - is it normal that builds from forks fail?

I see you opened a PR too, maybe just pick this change into your branch?

Reply:
test fail because `git clone` on Windows would use CRLF whereas the file in the zip file has just LF

Reply:
Tests from forks fail usually because of a test I setup to verify downloading files from private repos.
This test requires a GH access token which I configured using secrets,  and for security reasons, github doesn't share it with forks.

Reply:
I merged a PR to fix the test failure. The change just normalise the EOL character within tests.
The builds are passing now. Also I included the  generated `dist/index.js` with that. 
Thanks again for for sending a fix :)